src/Runner.Worker/ActionCommandManager.cs

```diff
@@ -75,6 +75,11 @@ public bool TryProcessCommand(IExecutionContext context, string input, Container
                 return false;
             }
 
+            if (!ActionCommandManager.EnhancedAnnotationsEnabled(context) && actionCommand.Command == "notice")
+            {
+                return false;
```

Comment:
We probably want a Trace or debug message here explaining we are skipping because the ff isn't enabled.

Reply:
Debug is user visible if they turn on the debug flag, so we can say something like `Skipping notice annotation as server does not support notice annotations`

Trace is for our purposes, so we can be a bit more detailed about what we say, mentioning FF's and the like.

Reply:
Debug is user visible if they turn on the debug flag, so we can say something like `Skipping notice annotation as server does not support notice annotations`

Trace is for our purposes, so we can be a bit more detailed about what we say, mentioning FF's and the like.